src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -55,8 +102,40 @@ def get_default_presto_profile_name():
     show_default=True,
```

Comment:
In the line above please use a lambda function so that `get_config_option` is executed lazily, otherwise we load configuration (and pydantic) upon each verdi invocation, including tab completion.

```suggestion
    default=lambda: get_config_option('autofill.user.email') or 'aiida@localhost',
    show_default=True,
```

Reply:
This does mean a slight change in the help text. If I have a default email setup, instead of 

```
  --email TEXT       Email of the default user.  [default: daniel.hollas@bristol.ac.uk]
```

it would print

```
  --email TEXT       Email of the default user.  [default: (dynamic)]
```

Since this cuts down the import time by 80ms I think it's a reasonable price to pay.

Reply:
Although I liked the default actually showing the custom default, it was being a pain in the ass because the pre-commit hook to auto generate the docs would change it each time, and each time I would have to ignore that change when committing. By putting it behind a lambda, that pain will be relieved 😅 so I will accept this request